sdcm/tester.py

```diff
@@ -358,6 +363,20 @@ def __init__(self, *args):  # pylint: disable=too-many-statements,too-many-local
         time.sleep(0.5)
         InfoEvent(message=f"TEST_START test_id={self.test_config.test_id()}").publish()
 
+    @property
+    def argus_test_run(self):
+        if not self._argus_test_run and self._use_argus:
+            try:
+                argus_test_run = ArgusTestRun.from_sct_config(test_id=UUID(self.test_id),
+                                                              test_module_path=self.test_config.test_name(),
+                                                              sct_config=self.params)
+                argus_test_run.save()
+                self._argus_test_run = argus_test_run
+            except Exception:  # pylint: disable=broad-except
+                self._use_argus = False
```

Comment:
let's return MagicMock() when we encountered exception

Reply:
Done